From 69aa6dffbbe5d3a97d4a8fd0cdfa9a440780598a Mon Sep 17 00:00:00 2001
From: Elia Devito <eliadevito@gmail.com>
Date: Mon, 10 Apr 2023 17:34:21 +0200
Subject: [PATCH] upgrade wxWidget to 3.2.2.1

---
 deps/wxWidgets/wxWidgets.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/deps/wxWidgets/wxWidgets.cmake b/deps/wxWidgets/wxWidgets.cmake
index bc1654f45..c6eb631a5 100644
--- a/deps/wxWidgets/wxWidgets.cmake
+++ b/deps/wxWidgets/wxWidgets.cmake
@@ -13,8 +13,8 @@ if (UNIX AND NOT APPLE) # wxWidgets will not use char as the underlying type for
 endif()
 
 prusaslicer_add_cmake_project(wxWidgets
-    URL https://github.com/prusa3d/wxWidgets/archive/0b49beaacce17d90f0c370ecd73221abd089667a.zip
-    URL_HASH SHA256=8fa978a76d6bd811b30eecc5124186b9ad54290b820f3a354e85bfa9dae6a5ce
+    URL https://github.com/wxWidgets/wxWidgets/archive/refs/tags/v3.2.2.1.tar.gz
+    URL_HASH SHA256=0b96807e700288652f1538d8f7db25559cbb890f15fe2001ca7eef6f7cce3262
     DEPENDS ${PNG_PKG} ${ZLIB_PKG} ${EXPAT_PKG} dep_TIFF dep_JPEG dep_NanoSVG
     CMAKE_ARGS
         -DwxBUILD_PRECOMP=ON
-- 
2.39.2

