From d5b190ef07cdec2f8f30f8e728b676d5cfe2aacf Mon Sep 17 00:00:00 2001
From: Elia Devito <eliadevito@gmail.com>
Date: Tue, 21 Feb 2023 15:32:30 +0100
Subject: [PATCH] don't override GDK_BACKEND env

---
 src/PrusaSlicer.cpp | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/PrusaSlicer.cpp b/src/PrusaSlicer.cpp
index c5bbccb24..4d9fce9e5 100644
--- a/src/PrusaSlicer.cpp
+++ b/src/PrusaSlicer.cpp
@@ -75,12 +75,12 @@ int CLI::run(int argc, char **argv)
     // Save the thread ID of the main thread.
     save_main_thread_id();
 
-#ifdef __WXGTK__
-    // On Linux, wxGTK has no support for Wayland, and the app crashes on
-    // startup if gtk3 is used. This env var has to be set explicitly to
-    // instruct the window manager to fall back to X server mode.
-    ::setenv("GDK_BACKEND", "x11", /* replace */ true);
-#endif
+// #ifdef __WXGTK__
+//     // On Linux, wxGTK has no support for Wayland, and the app crashes on
+//     // startup if gtk3 is used. This env var has to be set explicitly to
+//     // instruct the window manager to fall back to X server mode.
+//     ::setenv("GDK_BACKEND", "x11", /* replace */ true);
+// #endif
 
 	// Switch boost::filesystem to utf8.
     try {
-- 
2.39.2

