diff --git a/deps/+GLEW/GLEW.cmake b/deps/+GLEW/GLEW.cmake
index 529d1ae..e85bb53 100644
--- a/deps/+GLEW/GLEW.cmake
+++ b/deps/+GLEW/GLEW.cmake
@@ -7,5 +7,6 @@ add_cmake_project(
   EMSCRIPTEN_EXCLUDED ON
   CMAKE_ARGS
     -DBUILD_UTILS=OFF
+    -DGLEW_EGL=ON
     -DCMAKE_POLICY_VERSION_MINIMUM=3.5
 )
diff --git a/deps/+wxWidgets/wxWidgets.cmake b/deps/+wxWidgets/wxWidgets.cmake
index 4500b83..62abf84 100644
--- a/deps/+wxWidgets/wxWidgets.cmake
+++ b/deps/+wxWidgets/wxWidgets.cmake
@@ -48,7 +48,7 @@ add_cmake_project(wxWidgets
         -DwxUSE_LIBSDL=OFF
         -DwxUSE_STC=OFF
         -DwxUSE_XTEST=OFF
-        -DwxUSE_GLCANVAS_EGL=OFF
+        -DwxUSE_GLCANVAS_EGL=ON
         -DwxUSE_WEBREQUEST=OFF
         -DwxUSE_UNSAFE_WXSTRING_CONV=OFF
         -DwxUSE_EXCEPTIONS=OFF
