From b4bd399e4a967a0141eaccd915fe936354375096 Mon Sep 17 00:00:00 2001
From: Elia Devito <eliadevito@gmail.com>
Date: Thu, 31 Aug 2023 17:32:28 +0200
Subject: [PATCH] add tbb/parallel_for.h include to Measure.cpp

---
 src/libslic3r/Measure.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/libslic3r/Measure.cpp b/src/libslic3r/Measure.cpp
index 559336294..9602b14c1 100644
--- a/src/libslic3r/Measure.cpp
+++ b/src/libslic3r/Measure.cpp
@@ -4,9 +4,10 @@
 
 #include "libslic3r/Geometry/Circle.hpp"
 #include "libslic3r/SurfaceMesh.hpp"
-#include <numeric>
+
 
 #include <numeric>
+#include <tbb/parallel_for.h>
 
 #define DEBUG_EXTRACT_ALL_FEATURES_AT_ONCE 0
 
-- 
2.41.0

