From a03bff91c3c23a187099e3e51480b97f164d3f81 Mon Sep 17 00:00:00 2001
From: Elia Devito <eliadevito@gmail.com>
Date: Thu, 31 Aug 2023 18:06:34 +0200
Subject: [PATCH] add libslic3r/ClipperUtils.hpp to TextLines.cpp

---
 src/slic3r/GUI/TextLines.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/slic3r/GUI/TextLines.cpp b/src/slic3r/GUI/TextLines.cpp
index 8916ce5ae..bc3eb58a9 100644
--- a/src/slic3r/GUI/TextLines.cpp
+++ b/src/slic3r/GUI/TextLines.cpp
@@ -10,6 +10,7 @@
 
 #include "libslic3r/AABBTreeLines.hpp"
 #include "libslic3r/ExPolygonsIndex.hpp"
+#include "libslic3r/ClipperUtils.hpp"
 
 #include "slic3r/GUI/Selection.hpp"
 #include "slic3r/GUI/GLCanvas3D.hpp"
-- 
2.41.0

