From f0bd4db9bd7fec358a615af94a8517d906019c98 Mon Sep 17 00:00:00 2001
From: Elia Devito <eliadevito@gmail.com>
Date: Mon, 17 Jun 2024 17:30:41 +0200
Subject: [PATCH] don't build occt doc

---
 deps/+OCCT/OCCT.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/deps/+OCCT/OCCT.cmake b/deps/+OCCT/OCCT.cmake
index 5b7743e58..0ef431e84 100644
--- a/deps/+OCCT/OCCT.cmake
+++ b/deps/+OCCT/OCCT.cmake
@@ -20,4 +20,5 @@ add_cmake_project(OCCT
 		-DBUILD_MODULE_ModelingAlgorithms=OFF
 		-DBUILD_MODULE_ModelingData=OFF
 		-DBUILD_MODULE_Visualization=OFF
+        -DBUILD_DOC_Overview=OFF
 )
-- 
2.45.2

