From acd28c30bff4715b237eeb05a776a3aa97de61fb Mon Sep 17 00:00:00 2001
From: Elia Devito <eliadevito@gmail.com>
Date: Thu, 31 Aug 2023 16:55:49 +0200
Subject: [PATCH] add boost/log/trivial.hpp to CutUtlis.cpp

---
 src/libslic3r/CutUtils.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/libslic3r/CutUtils.cpp b/src/libslic3r/CutUtils.cpp
index 1622af87e..0144f8a0b 100644
--- a/src/libslic3r/CutUtils.cpp
+++ b/src/libslic3r/CutUtils.cpp
@@ -7,6 +7,7 @@
 #include "TriangleSelector.hpp"
 #include "ObjectID.hpp"
 
+#include <boost/log/trivial.hpp>
 
 namespace Slic3r {
 
-- 
2.41.0

