From cefef527525af47387841b5d11b6fecc8d08822b Mon Sep 17 00:00:00 2001
From: Elia Devito <eliadevito@gmail.com>
Date: Fri, 24 Feb 2023 18:00:01 +0100
Subject: [PATCH] upgrade wxWidget to 3.2.2.1

---
 deps/wxWidgets/wxWidgets.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/deps/wxWidgets/wxWidgets.cmake b/deps/wxWidgets/wxWidgets.cmake
index 214a38139..b69a01edb 100644
--- a/deps/wxWidgets/wxWidgets.cmake
+++ b/deps/wxWidgets/wxWidgets.cmake
@@ -13,8 +13,8 @@ if (UNIX AND NOT APPLE) # wxWidgets will not use char as the underlying type for
 endif()
 
 prusaslicer_add_cmake_project(wxWidgets
-    URL https://github.com/prusa3d/wxWidgets/archive/4fd2120c913c20c3bb66ee9d01d8ff5087a8b90a.zip
-    URL_HASH SHA256=5b59e8b4dccf73e109c6588f6a69bcfe4e02e930af53c43d5d1329c1f3d83ec9
+    URL https://github.com/wxWidgets/wxWidgets/archive/refs/tags/v3.2.2.1.tar.gz
+    URL_HASH SHA256=0b96807e700288652f1538d8f7db25559cbb890f15fe2001ca7eef6f7cce3262
     DEPENDS ${PNG_PKG} ${ZLIB_PKG} ${EXPAT_PKG} dep_TIFF dep_JPEG dep_NanoSVG
     CMAKE_ARGS
         -DwxBUILD_PRECOMP=ON
-- 
2.39.2

