From 52bbe021de3495eeef5ca8ddbb4d0795d5d332cd Mon Sep 17 00:00:00 2001
From: Elia Devito <eliadevito@gmail.com>
Date: Sat, 22 Jun 2024 15:39:15 +0200
Subject: [PATCH] add missing include

---
 src/slic3r/GUI/RulerForDoubleSlider.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/slic3r/GUI/RulerForDoubleSlider.cpp b/src/slic3r/GUI/RulerForDoubleSlider.cpp
index d1d6f2f77..7b2bd4023 100644
--- a/src/slic3r/GUI/RulerForDoubleSlider.cpp
+++ b/src/slic3r/GUI/RulerForDoubleSlider.cpp
@@ -2,8 +2,12 @@
 ///|/
 ///|/ PrusaSlicer is released under the terms of the AGPLv3 or higher
 ///|/
+#include <algorithm>
+#include <cmath>
+
 #include "RulerForDoubleSlider.hpp"
 #include "libslic3r/CustomGCode.hpp"
+#include "libslic3r/Point.hpp"
 
 using namespace Slic3r;
 using namespace CustomGCode;
-- 
2.45.2

