From 5d3b7e461111b460dc403e62d090c392380034b9 Mon Sep 17 00:00:00 2001
From: Elia Devito <eliadevito@gmail.com>
Date: Thu, 31 Aug 2023 17:52:29 +0200
Subject: [PATCH 1/3] add boost/log/trivial.hpp to CutUtlis.cpp

---
 src/libslic3r/CutUtils.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/libslic3r/CutUtils.cpp b/src/libslic3r/CutUtils.cpp
index 1622af87e1..0144f8a0ba 100644
--- a/src/libslic3r/CutUtils.cpp
+++ b/src/libslic3r/CutUtils.cpp
@@ -7,6 +7,7 @@
 #include "TriangleSelector.hpp"
 #include "ObjectID.hpp"
 
+#include <boost/log/trivial.hpp>
 
 namespace Slic3r {
 

From fca573854b1d0391c378c3b22a3ad3559fc2e5df Mon Sep 17 00:00:00 2001
From: Elia Devito <eliadevito@gmail.com>
Date: Thu, 31 Aug 2023 17:53:01 +0200
Subject: [PATCH 2/3] add tbb/parallel_for.h include to Measure.cpp

---
 src/libslic3r/Measure.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/libslic3r/Measure.cpp b/src/libslic3r/Measure.cpp
index 5593362949..9602b14c11 100644
--- a/src/libslic3r/Measure.cpp
+++ b/src/libslic3r/Measure.cpp
@@ -4,9 +4,10 @@
 
 #include "libslic3r/Geometry/Circle.hpp"
 #include "libslic3r/SurfaceMesh.hpp"
-#include <numeric>
+
 
 #include <numeric>
+#include <tbb/parallel_for.h>
 
 #define DEBUG_EXTRACT_ALL_FEATURES_AT_ONCE 0
 

From 8be3b592b438bf3d2213fedcea2a967e179b403b Mon Sep 17 00:00:00 2001
From: Elia Devito <eliadevito@gmail.com>
Date: Thu, 31 Aug 2023 18:06:34 +0200
Subject: [PATCH 3/3] add libslic3r/ClipperUtils.hpp to TextLines.cpp

---
 src/slic3r/GUI/TextLines.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/slic3r/GUI/TextLines.cpp b/src/slic3r/GUI/TextLines.cpp
index 8916ce5ae4..bc3eb58a91 100644
--- a/src/slic3r/GUI/TextLines.cpp
+++ b/src/slic3r/GUI/TextLines.cpp
@@ -10,6 +10,7 @@
 
 #include "libslic3r/AABBTreeLines.hpp"
 #include "libslic3r/ExPolygonsIndex.hpp"
+#include "libslic3r/ClipperUtils.hpp"
 
 #include "slic3r/GUI/Selection.hpp"
 #include "slic3r/GUI/GLCanvas3D.hpp"
