diff --git a/deps/+Blosc/Blosc.cmake b/deps/+Blosc/Blosc.cmake
index 0bb5b055e6..a07695c182 100644
--- a/deps/+Blosc/Blosc.cmake
+++ b/deps/+Blosc/Blosc.cmake
@@ -14,6 +14,8 @@ endif ()
 
 # Common CMake arguments for Blosc
 set(_blosc_cmake_args
+        -DCMAKE_C_FLAGS="-std=gnu17"
+        -DCMAKE_CXX_FLAGS="-std=gnu++17"
         -DCMAKE_POSITION_INDEPENDENT_CODE=ON
         -DBUILD_SHARED=${_build_shared}
         -DBUILD_STATIC=${_build_static}
@@ -36,4 +38,4 @@ add_cmake_project(Blosc
         EMSCRIPTEN_EXCLUDED OFF
         CMAKE_ARGS
         ${_blosc_cmake_args}
-)
\ No newline at end of file
+)
