diff --git a/deps/+Blosc/Blosc.cmake b/deps/+Blosc/Blosc.cmake
index ebd6b3d176..568427485f 100644
--- a/deps/+Blosc/Blosc.cmake
+++ b/deps/+Blosc/Blosc.cmake
@@ -16,6 +16,8 @@ add_cmake_project(Blosc
     URL https://github.com/Blosc/c-blosc/archive/8724c06e3da90f10986a253814af18ca081d8de0.zip
     URL_HASH SHA256=53986fd04210b3d94124b7967c857f9766353e576a69595a9393999e0712c035
     CMAKE_ARGS
+        -DCMAKE_C_FLAGS="-std=gnu17"
+        -DCMAKE_CXX_FLAGS="-std=gnu++17"
         -DCMAKE_POSITION_INDEPENDENT_CODE=ON
         -DBUILD_SHARED=${_build_shared}
         -DBUILD_STATIC=${_build_static}
