diff --git a/deps/TBB/TBB.cmake b/deps/TBB/TBB.cmake
index 7020e43..11ba709 100644
--- a/deps/TBB/TBB.cmake
+++ b/deps/TBB/TBB.cmake
@@ -1,7 +1,10 @@
+set(_patch_command ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_LIST_DIR}/GNU.cmake cmake/compilers/GNU.cmake)
+
 prusaslicer_add_cmake_project(
     TBB
     URL "https://github.com/oneapi-src/oneTBB/archive/refs/tags/v2021.5.0.zip"
     URL_HASH SHA256=83ea786c964a384dd72534f9854b419716f412f9d43c0be88d41874763e7bb47
+    PATCH_COMMAND ${_patch_command}
     CMAKE_ARGS          
         -DTBB_BUILD_SHARED=OFF
         -DTBB_TEST=OFF
@@ -9,6 +12,8 @@ prusaslicer_add_cmake_project(
         -DCMAKE_DEBUG_POSTFIX=_debug
 )
 
+add_compile_definitions(__TBB_DYNAMIC_LOAD_ENABLED=0)
+
 if (MSVC)
     add_debug_dep(dep_TBB)
 endif ()
